app/src/views/GoogleLogin.jsx

```diff
@@ -0,0 +1,44 @@
+import React from 'react';
+import { Button, StyleSheet, Text, View } from 'react-native';
+import { AuthSession } from 'expo';
+
+const Google_APP_ID = '847385043058-1v26bjo0eahcc7sal4f86s7ed04mtjro.apps.googleusercontent.com';
```

Comment:
Should this value be private? If so we really don't want it to go on github due to bots web scraping github

Reply:
yeah Im about to make it private now. I forgot about that lol